packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -607,6 +640,12 @@ const MultiSelect = (props: Props) => {
     const filteredItems = getMenuItems(allChildren);
     const isDisabled = numEnabledOptions === 0 || disabled;
 
+    React.useEffect(() => {
+        if (open) {
+            handleAnnouncement(someSelected(filteredItems.length));
```

Comment:
I'm realizing now that the announcement for when the number of filtered items changes is very similar to when the the selected values are updated!
- When filtered items change, the message is: `numSelectedValues === 1 ? "1 item" : ${numSelectedValues} items`
- When selected values change, the message is the opener text. If one item is selected, the opener text is the label of the item. If multiple are selected, the opener text is also in the format `x items`

It could be difficult to tell the difference between these 2 cases. This could be a future improvement though since we're working with the existing labels!

Reply:
Yes, good point! This does depend on the labels, which have to be localized. It would be helpful to tackle that in a later phase if possible.